packages/nimble-components/src/table/index.ts

```diff
@@ -439,11 +417,7 @@ export class Table<
 
     /** @internal */
     public handleCollapseAllGroupRows(): void {
```

Comment:
I didn't trace where this is used, but can you double check whether it should be renamed not to include "group" in its name as you did for other APIs?